### PR TITLE
CalculateTotalPrice created and added to relevant places.

### PR DESCRIPTION
**Pull request by davwel1997**
Still an issue with price being a text imput, which renders the "sum" of texts rather than total price.

### CODE REVIEW

File: client/src/components/order/shoppingCart/shoppingCart.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ class ShoppingCart extends Component {
         this.getSubmitDisabled = this.getSubmitDisabled.bind(this);
         this.submitOrder = this.submitOrder.bind(this);
     }
-
+    /*
     //Calculates the totalprice of the order
     calculateTotalPrice() {
         let totalPrice = 0;
```
Comment (steget) [inline client/src/components/order/shoppingCart/shoppingCart.js:20]: Remove completely

File: client/src/components/order/shoppingCart/shoppingCart.js [hunk 2]
```diff
@@ -112,8 +113,8 @@ class ShoppingCart extends Component {
                                         disabled={this.getSubmitDisabled()}
                                         onClick={this.submitOrder}
                                     >
-                                        Beställ - {this.calculateTotalPrice()}
-                                        kr
+                                        Beställ -{" "}
+                                        {this.props.calculateTotalPrice} kr
```
Comment (steget) [inline client/src/components/order/shoppingCart/shoppingCart.js:117]: Weird js insertion here for the space.
Replace with a normal space and let Prettier autoformat :)
```suggestion
                                        Beställ - 
                                        {this.props.calculateTotalPrice} kr
```

File: client/src/components/order/index.js [hunk 3]
```diff
@@ -128,6 +131,19 @@ class Order extends Component {
         return this.showMenu();
     }
 
+    calculateTotalPrice() {
+        if (this.state.cartItems.length === []) {
```
Comment (steget) [inline client/src/components/order/index.js:135]: Why compare with an empty array? Checking if length is equal to 0 makes more sense

File: client/src/components/order/index.js [hunk 4]
```diff
@@ -128,6 +131,19 @@ class Order extends Component {
...
+            return 0;
+        } else {
+            let tempCalc = 0;
+            var i;
```
Comment (steget) [inline client/src/components/order/index.js:139]: Try to use let/const in the entire application

File: client/src/components/order/index.js (hunk 3 above)
Comment (davwel1997) [reply client/src/components/order/index.js:135]: Got a lot of errors when comparing lengths, undefined, null, etc. This was the only one that worked for some reason

**Pull request merged**